### DISCUSSION TITLE
Migration namespace

### DISCUSSION THREAD

**Discussion by pleone** [nWidart/laravel-modules#1203] [Q&A]
Hi!
Quick question: is it on purpose that migration files have no namespace in it when generated with the artisan command? 
I know it's unlikely to happen, but there might be issues with multiple migrations sharing the same classname.

Thanks @nWidart for the awesome work 🚀

**Comment by maxcelos** [comment]
I guess this package just replicates the Laravel implementations to a modular format. Native Laravel migrations has no namespace either. 

This question should be asked on Laravel repository.
